frontend/src/graphql-components/apolloClientSetup.ts

```diff
@@ -0,0 +1,26 @@
+import { ApolloClient, createHttpLink, InMemoryCache } from '@apollo/client';
+import { setContext } from '@apollo/client/link/context';
+import { REACT_APP_CROWNLABS_GRAPHQL_URL } from '../env';
+
+const TOKEN = '';
+
+const httpLink = createHttpLink({
+  uri: REACT_APP_CROWNLABS_GRAPHQL_URL,
+});
+
+const authLink = setContext((_, { headers }) => {
+  const token = TOKEN;
+  return {
+    headers: {
+      ...headers,
+      authorization: token ? `Bearer ${token}` : '',
+    },
+  };
+});
```

Comment:
I'm not really understanding what's going on here, what's that empty constant token for?

Reply:
I guess it is temporary, for development purposes, because we don't have auth in the app, hence no way to obtain the token